### PR TITLE
Udp adapter for logger

### PR DESCRIPTION
**Pull request by vitalypanait**

### CODE REVIEW

File: Library/Phalcon/Logger/Adapter/Udplogger.php [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+        if ($this->socket === null) {
+            $this->socket = socket_create(AF_INET, SOCK_DGRAM, SOL_UDP);
+        }
+
+        socket_sendto($this->socket, $message, strlen($message), 0, $this->options['url'], $this->options['port']);
+
+        $this->logs = [];
+    }
+}
```
Comment (sergeyklay) [inline Library/Phalcon/Logger/Adapter/Udplogger.php]: > Expected 1 newline at end of file; 0 found

Could you please fix this PSR warning?

Comment (sergeyklay) [inline Library/Phalcon/Logger/Adapter/Udplogger.php]: selection_153

Comment (vitalypanait) [inline Library/Phalcon/Logger/Adapter/Udplogger.php]: @sergeyklay done

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**